packages/graphql/tests/tck/connections/projections/projections.test.ts

```diff
@@ -159,10 +159,15 @@ describe("Relay Cursor Connection projections", () => {
             WHERE this.title = $param0
             CALL (this) {
                 MATCH (this)<-[this0:ACTED_IN]-(this1:Actor)
-                WITH count(this1) AS totalCount
-                RETURN { totalCount: totalCount } AS var2
+                WITH collect({ node: this1, relationship: this0 }) AS edges, count(this1) AS totalCount
+                CALL (edges) {
+                    UNWIND edges AS edge
+                    WITH edge.node AS this1, edge.relationship AS this0
+                    RETURN collect({ node: { __id: id(this1), __resolveType: \\"Actor\\" } }) AS var2
+                }
+                RETURN { edges: var2, totalCount: totalCount } AS var3
             }
-            RETURN this { .title, actorsConnection: var2 } AS this"
+            RETURN this { .title, actorsConnection: var3 } AS this"
```

Comment:
I'm not sure why this has changed, as it does seem to be the opposite of what the issue is about. Was this another separate bug?